sentry-kotlin-multiplatform/src/commonJvmMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryOptionsExtensions.kt

```diff
@@ -39,4 +43,9 @@ internal fun JvmSentryOptions.applyJvmBaseOptions(options: SentryOptions) {
     this.setBeforeBreadcrumb { jvmBreadcrumb, _ ->
         options.beforeBreadcrumb?.invoke(jvmBreadcrumb.toKmpBreadcrumb())?.toJvmBreadcrumb()
     }
+    this.setBeforeSend { jvmSentryEvent, hint ->
+        options.beforeSend?.invoke(SentryEvent(jvmSentryEvent))?.let {
+            jvmSentryEvent.applyKmpEvent(it)
```

Comment:
in all cases we don't want to replace the event but only modify / apply changes to it